package.json

```diff
@@ -140,9 +153,9 @@
     "apollo-server-express": "3.10.2",
     "apollo-server-plugin-response-cache": "3.8.2",
     "archiver": "6.0.1",
-    "aws-elasticsearch-connector": "9.0.0",
-    "aws-sdk": "2.814.0",
-    "aws4": "1.11.0",
+    "aws-elasticsearch-connector": "9.2.0",
+    "aws-sdk": "2.1651.0",
+    "aws4": "1.13.0",
```

Comment:
~Was hoping this PR managed to refactor away some of these alternate packages. 🙃

Reply:
- `'aws-sdk'` isn't imported anywhere anymore; could be removed
- `'aws4`' is still used in `kibana.service.ts` for signing; could be refactored, but the project owner would have to do that (too much application context needed)

Reply:
Tried removing `aws-sdk` package, but it's being used by `winston-cloudwatch`. We need to replace it anyway. Strange, though, that `winston-cloudwatch` doesn't pull in `aws-sdk` itself as a dependency :confused: